airflow/settings.py

```diff
@@ -118,6 +118,27 @@
     "upstream_failed": "orange",
 }
 
+# list of the environment variables that need to be masked from the logs
+# some of these could be sensitive and there's no reason to log those
+ENV_VARIABLES_TO_MASK = [
+    "AIRFLOW__CORE__FERNET_KEY",
+    "AIRFLOW__SMTP__SMTP_PORT",
+    "AIRFLOW__SMTP__SMTP_PASSWORD",
```

Comment:
This list feels wrong to hard code. I'm fairly sure there is already a concept in the configuration of a "sensitive" config which we should reuse.

Reply:
Yeah: 
- https://github.com/apache/airflow/blob/aaa5f3566663eb3e9a0e7bf0522d533fbb1010d5/airflow/config_templates/config.yml#L182
- https://github.com/apache/airflow/blob/main/airflow/configuration.py#L297-L323

Reply:
Also. Masking the PORT? Why?!

Reply:
Well yeah that makes sense, let me consume those and try to mask those.